src/utils/transformBundle.js

```diff
@@ -0,0 +1,28 @@
+export default function transformBundle ( source, transformers ) {
+	if ( typeof source === 'string' ) {
+		source = {
+			code: source,
+			map: null
+		};
+	}
+
+	return transformers.reduce( ( previous, transformer ) => {
+		let result = transformer( previous );
```

Comment:
Module transformers are called with the code from the previous step, rather than a `{code, map}` object – I think we should stick to that, because it makes transformers a lot more straightforward. Somehow we need to figure out how to compose all the sourcemaps together :grin: